src/models/Pool.ts

```diff
@@ -0,0 +1,22 @@
+import {index, modelOptions, prop, Severity} from '@typegoose/typegoose';
+
+@index({address: 1})
+@index({token0: 1})
+@index({token1: 1})
+@modelOptions({options: {allowMixed: Severity.ALLOW}})
+export class Pool {
```

Comment:
I think this is too generic name
```suggestion
export class SovrynPool {
```